.byebug_history

```diff
@@ -0,0 +1 @@
+@auth.provided? and @auth.basic? and @auth.credentials and @auth.credentials == [ENV['METRICS_API_USERNAME'], ENV['METRICS_API_PASSWORD']]
```

Comment:
ignore this file

Reply:
it's already in the gitignore but it won't go away -https://github.com/theodi/bothan/blob/master/.gitignore#L14
 I must have committed it by accident at some point, hope the force delete purges it